### PR TITLE
Fixing simple regression dialogue and ucrReceiver

### PR DESCRIPTION
**Pull request by stevekogo**
Fixed dlgRegressionSimple  and implemented Shift + Tab on ucrReceiver

### CODE REVIEW

File: instat/dlgRegressionSimple.Designer.vb [hunk 1]
```diff
@@ -1,192 +1,111 @@
...
+        Me.txtResidualMsg = New System.Windows.Forms.TextBox()
+        Me.txtdf = New System.Windows.Forms.TextBox()
+        Me.lblResponse = New System.Windows.Forms.Label()
+        Me.lblExplanatory = New System.Windows.Forms.Label()
+        Me.lblFittedModel = New System.Windows.Forms.Label()
+        Me.txtLastLM = New System.Windows.Forms.TextBox()
+        Me.cmdRegressionOptions = New System.Windows.Forms.Button()
+        Me.ucrExplanatory = New instat.ucrReceiverSingle()
+        Me.ucrResponse = New instat.ucrReceiverSingle()
+        Me.UcrSelectorByDataFrameAddRemoveSimpleReg = New instat.ucrSelectorByDataFrameAddRemove()
```
Comment (dannyparsons) [inline instat/dlgRegressionSimple.Designer.vb]: The name of "UcrSelectorByDataFrameAddRemoveSimpleReg" should have a lower case "u". And a shorter name would be better if possible.

File: instat/dlgRegressionSimple.vb [hunk 2]
```diff
@@ -1,50 +1,98 @@
...
+
+    Private Sub dlgRegressionSimple_Load(sender As Object, e As EventArgs) Handles Me.Load
+        ucrBase.clsRsyntax.SetFunction("lm")
+        ucrBase.clsRsyntax.iCallType = 2
+        ucrResponse.Selector = UcrSelectorByDataFrameAddRemoveSimpleReg
+
+        ucrExplanatory.Selector = UcrSelectorByDataFrameAddRemoveSimpleReg
+        autoTranslate(Me)
+        'To add correct help ID
+        ucrBase.iHelpTopicID = 171
```
Comment (dannyparsons) [inline instat/dlgRegressionSimple.vb]: It's better to comment out or remove this line if you don't have the Help ID yet. 
By default if the Help ID is not set then the help still goes to the main table of contents. That's better than it going to the wrong page until we have a help page for it.

File: instat/dlgRegressionSimple.vb [hunk 3]
```diff
@@ -1,50 +1,98 @@
...
+    End Sub
+
+    Private Sub TestOKEnabled()
+        Dim strResponse As String = ""
+        Dim strExplanatory As String = ""
+        strResponse = ucrResponse.GetVariableNames(bWithQuotes:=False)
+        strExplanatory = ucrExplanatory.GetVariableNames(bWithQuotes:=False)
+
+        If ((Not (strResponse = "")) And (Not (strExplanatory = ""))) Then
+            ucrBase.clsRsyntax.AddParameter("formula", strResponse & "~" & strExplanatory)
```
Comment (dannyparsons) [inline instat/dlgRegressionSimple.vb]: @volloholic should this be done using an Operator now? Could someone show Steve how these work.

File: instat/dlgRegressionSimple.vb [hunk 4]
```diff
@@ -1,50 +1,98 @@
...
+
+    Private Sub cmdRegressionOptions_Click(sender As Object, e As EventArgs) Handles cmdRegressionOptions.Click
+        ' sdgSimpleRegressionOptions.ShowDialog()
+    End Sub
+
+    Private Sub ucrResponse_SelectionChanged() Handles ucrResponse.SelectionChanged
+        'Test ok enabled
+        TestOKEnabled()
+    End Sub
+    Private Sub ucrExplanatory_SelectionChanged() Handles ucrExplanatory.SelectionChanged
```
Comment (dannyparsons) [inline instat/dlgRegressionSimple.vb]: Shouldn't there be some parameter added when the selection changes here and for ucrResponse?

File: instat/dlgRegressionSimple.vb (hunk 2 above)
Comment (dannyparsons) [inline instat/dlgRegressionSimple.vb]: If this is the correct ID already then you can just remove the comment.

**Comment by stevekogo** [comment]
@dannyparsons I have made a commit with the changes effected. Could you please go trough it.

**Comment by dannyparsons** [comment]
Great, looks good now.

**Pull request merged**